[2024-04-15T09:06:08.350+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.extract_data_from_csv scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T09:06:08.374+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.extract_data_from_csv scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T09:06:08.375+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-15T09:06:08.398+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data_from_csv> on 2024-04-14 00:00:00+00:00
[2024-04-15T09:06:08.408+0000] {standard_task_runner.py:60} INFO - Started process 318 to run task
[2024-04-15T09:06:08.412+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'extract_data_from_csv', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '1395', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmptjdii30r']
[2024-04-15T09:06:08.414+0000] {standard_task_runner.py:88} INFO - Job 1395: Subtask extract_data_from_csv
[2024-04-15T09:06:08.474+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_data.extract_data_from_csv scheduled__2024-04-14T00:00:00+00:00 [running]> on host 41da1b14e620
[2024-04-15T09:06:08.569+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='extract_data_from_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-15T09:06:08.572+0000] {logging_mixin.py:188} INFO - Folder ../data/tolldata does not exist.
[2024-04-15T09:06:08.573+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-15T09:06:08.994+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=extract_data_from_csv, execution_date=20240414T000000, start_date=20240415T090608, end_date=20240415T090608
[2024-04-15T09:06:09.068+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-15T09:06:09.119+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
